controls/pcidss_4.yml

```diff
@@ -2787,7 +2787,6 @@ controls:
       rules:
         - var_auditd_name_format=fqd
         - auditd_name_format
-        - audit_profile_set
```

Comment:
Huh - interesting... nice catch. Looks like this slipped in with https://github.com/ComplianceAsCode/content/pull/11214

Tagging @marcusburghardt and @Mab879 for awareness since this relates to a RHEL profile (but it shouldn't affect anything, I don't think?)

Reply:
I quick grep of the build product for RHEL9 reveals that this removal should be fine.